lib/routes/podcast/boost.dart

```diff
@@ -235,4 +289,15 @@ class _BoostWidgetState extends State<BoostWidget> {
               });
         });
   }
+
+  static int _getClosestBoostAmount(int customAmount, List presetAmountList,
```

Comment:
Can't we use this function also for the equal case? This way we won't need to split the condition depends on whether the custom amount is taken from the list or not.
Also I would make this function more readable by giving each case its own line, it is OK if it will be a little bit longer IMO.

Reply:
We can split this into _getNextAmount and _getPreviousAmount methods and get rid of the condition and index logic.